examples/http-load-balancer/http_load_balancer.description

```diff
@@ -1 +1 @@
-// With the LoadBalancer, Ballerina users can configure multiple connectors to distribute load across them with desired load balancing strategy.
+// Load balancers help distribute the load when multiple connectors are configured in Ballerina.
```

Comment:
@VijithaEkanayake : Can you check and approve, please?

Reply:
@shavidissa Shall we rename connectors to target endpoints??

Reply:
Ack! Updated the description.